viadot/tasks/mindful.py

```diff
@@ -117,12 +117,13 @@ def run(
                 credentials_mindful = None
                 raise CredentialError("Credentials not found.")
 
-        header = {
-            "Authorization": f"Bearer {credentials_mindful.get('VAULT')}",
-        }
+        auth = (
+            credentials_mindful["CUSTOMER_UUID"],
+            credentials_mindful["AUTH_TOKEN"],
```

Comment:
Here you are getting `CUSTOMER_UUID` and `AUTH_TOKEN` but docstring in the source says that `The structure is user and password`. What is correct?

Reply:
In mindful, the user is defined by an ID, in this case, `CUSTOMER_UUID`, and the password is defined by, what they said `AUTH TOKEN`, but this is a password, I  mean, if you need to generate a token for OAuth, this not would be correct, you would have to create a token.

Reply:
when I saved it to the credentials, I added the same words they decided to use for the user (mindful velux user) and the password (mindful velux password).